### PR TITLE
Introduce skroutza.js

### PR DESCRIPTION
**Pull request by mrwhizzy**

### CODE REVIEW

**Review by MrGoumX — APPROVED** [review]
+1 LGTM! 🚀 💯

**Review by alexunder193 — APPROVED** [review]

File: Gruntfile.coffee [hunk 1]
```diff
@@ -356,6 +356,14 @@ module.exports = (grunt) ->
         src: 'plugins/*'
         dest: '.tmp/js'
 
+      services:
```
Comment (harrisred) [inline Gruntfile.coffee:359]: [minor] What if we rename this to `alias` or `skroutza`? 🤔

Comment (mrwhizzy) [reply Gruntfile.coffee:359]: good point, this was a leftover name to be honest

**Review by harrisred — APPROVED** [review]
LGTM 🚀

**Pull request merged**